ostree-upload-server.py

```diff
@@ -76,10 +85,27 @@ def get_queue(self):
     def join(self, timeout=None):
         return self.queue.join(timeout)
 
+class Counter:
+    def __init__(self):
+        self.count = 0
+
+    def __enter__(self):
+        self.count += 1
+        print("counter now " + str(self.count))
+        return self.count
+
+    def __exit__(self, type, value, traceback):
+        self.count -= 1
+        print("counter now " + str(self.count))
 
 app = Flask(__name__)
-app.config["UPLOAD_FOLDER"] = tempfile.mkdtemp("upload")
+tempdir = tempfile.mkdtemp("upload")
```

Comment:
Set the prefix instead of suffix. Also make it a bit more than just `upload` since it's ambiguous.